### PR TITLE
Docs tests

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Fixed tickets | #43
| License       | MIT

Follow up to #57 that adds docs build to travis.

### CODE REVIEW

**Comment by kunicmarko20** [comment]
If #57 gets merged, this should be merged after and only then.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@
 # PHPUnit
 /build
 /phpunit.xml
+
+# Documentation
+/docs/_build/*
```
Comment (ElectricMaxxx) [inline .gitignore]: this is doc/ now (pls. rebase on master)

Comment (kunicmarko20) [reply .gitignore]: still not merged

Comment (kunicmarko20) [reply .gitignore]: also this can't be merged before #57 and we still didn't agree if we will transfer to scripts

**Pull request merged**